packages/elementary_tools/elementary_cli/lib/generate/generate.dart

```diff
@@ -90,7 +90,14 @@ abstract class TemplateGeneratorCommand extends Command<void> {
         'script entry has no absolute path',
       );
     }
-    final defaultDirPath = p.join(p.dirname(Platform.script.path), templatesRelativeToExecutableDirectory);
+
+    final defaultDirPath = p.join(
+      p.dirname(p.joinAll(Platform.script.pathSegments)),
+      p.normalize(templatesRelativeToExecutableDirectory),
+    );
+    //ignore: avoid_print
```

Comment:
I thing the better way to use debugPrint or if (kDebugMode) instead ignoring

Reply:
This print was left by accident, now it's removed